### DISCUSSION TITLE
Welcome to openml-python Discussions!

### DISCUSSION THREAD

**Discussion by PGijsbers** [openml/openml-python#1164] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by varshneydevansh** [comment]
Hi everyone,

I am Devansh from India, live close to New Delhi. I opted for Computer Science because of the movie Iron Man back in my school, particularly the part of Jarvis really hooked me in. Hence, learned somewhat of ML/AI stuff during my college days. The pandemic has allowed me a time to restart everything again as I was leaving things incomplete and hence I started doing what I left without even trying is open-source contribution.

I looked at what openml is doing and loved the idea of organizing the ML research, datasets and related stuff with easy to access for everyone. I am a generalist who is eager to contribute and acquire new knowledge.

Thanks.

**Reply by PGijsbers** [reply]
Welcome! Lots of opportunity for that here :)
